### PR TITLE
chore(py38): remove unused dependencies

### PR DESCRIPTION
**Pull request by joshuarli**
setproctitle was used back when we ran gunicorn. Now that we're on uwsgi, it's not needed.

sqlparse can in theory be moved to requirements-dev.txt after I removed that utils file which wasn't being used, but it has to stay for now due to sentry.testutils.

werkzeug is completely unused.

pytest-html: We're pretty behind on versions here but anyway I don't believe anyone makes use of the HTML reports it generates, which can get quite large and unwieldy the more tests you run. Someone would have noticed by now because we're very inconsistent here, it's just acceptance tests.

If someone complains about pytest-html I'll just add it back no big deal, easier to ask for forgiveness not permission here.

### CODE REVIEW

**Comment by EvanPurkhiser** [comment]
> pytest-html: We're pretty behind on versions here but anyway I don't believe anyone makes use of the HTML reports it generates, which can get quite large and unwieldy the more tests you run. Someone would have noticed by now because we're very inconsistent here, it's just acceptance tests.

Could zeus have been using this?

**Comment by joshuarli** [comment]
> > pytest-html: We're pretty behind on versions here but anyway I don't believe anyone makes use of the HTML reports it generates, which can get quite large and unwieldy the more tests you run. Someone would have noticed by now because we're very inconsistent here, it's just acceptance tests.
> 
> Could zeus have been using this?

It was uploaded to zeus before, probably wasn't parsed for anything though. Besides I don't believe we upload to zeus anymore.

File: Makefile [hunk 1]
```diff
@@ -128,7 +128,7 @@ fetch-release-registry:
 
 run-acceptance:
 	@echo "--> Running acceptance tests"
-	py.test tests/acceptance --cov . --cov-report="xml:.artifacts/acceptance.coverage.xml" --junit-xml=".artifacts/acceptance.junit.xml" --html=".artifacts/acceptance.pytest.html" --self-contained-html
```
Comment (EvanPurkhiser) [inline Makefile:131]: @billyvg did we need this for anything?

Comment (billyvg) [reply Makefile:131]: Yeah at once point we uploaded this to zeus, we no longer upload anything to zeus. It's useful when acceptance shits the bed in CI (and not locally) and we have to debug, but I don't think it's something we use consistently

**Review by billyvg — APPROVED** [review]
I think if anyone needs `pytest-html` they can install it as a one-off, but doesn't seem something we need to keep in the repo long-term.

**Pull request merged**